packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -584,6 +593,10 @@ export class Cluster extends ClusterBase {
       }
     }
 
+    if (props.availabilityZoneRelocation && !nodeType.startsWith('ra3')) {
+      throw new Error(`Availability zone relocation is supported for only RA3 node types, got: ${props.nodeType}`);
+    }
+
     this.cluster = new CfnCluster(this, 'Resource', {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Ensure that user activity logging is enabled for the Redshift cluster. This feature logs each query before it is executed on the cluster&#39;s database. To activate this, associate a Redshift Cluster Parameter Group with the `enable_user_activity_logging` parameter set to `true`.

Reply:
I believe this feedback is inappropriate. This implementation is for existing L2 code, and to avoid breaking changes, we should not forcefully enable `enable_user_activity_logging`.